### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by AJTDaedalus**
User me123@gmail.com with password 12345678 is still created with admin privileges.

Please login with this user, go to the admin page, and confirm you can do the following:
Create a new role
Assign a role to a user
Remove a role from a user
Assign multiple roles from a user
Remove multiple roles from a user

Please test this both on the development server (python main.py) and via the docker production server.

### CODE REVIEW

**Comment by AJTDaedalus** [comment]
There is one new dependency that I've already added to requirements.txt, but to run locally using your environment use:
pip install WTForms-SQLAlchemy

File: code/app/auth/views.py [hunk 1]
```diff
@@ -67,6 +67,46 @@ def login():
...
+    if remove.remove.data and remove.validate():
+        for u in remove.user.iter_choices():
+            if u[2] == True:
+                user = User.query.get(u[0])
+                for r in remove.rem_roles.iter_choices():
+                    if r[2] == True:
+                        print(r[1])
+                        user.remove_role(r[1])
+
+    return render_template('security/admin.html', nrole=NewRoleForm(),
```
Comment (LSMachuca) [inline code/app/auth/views.py:107]: You are referencing `security/admin.html` but it does not exist. I get Error: `jinja2.exceptions.TemplateNotFound: security/admin.html`
Did you miss adding the files in the `security` directory?

Comment (AJTDaedalus) [reply code/app/auth/views.py:107]: That's exactly what happened!  I've added it and pushed to origin, it's there now.

Comment (AJTDaedalus) [reply code/app/auth/views.py:107]: FYI another branch just merged into master and I updated to allow the merge, so it's worth a double check on my content.

Comment (LSMachuca) [reply code/app/auth/views.py:107]: Was working before the merge. Not sure why is not working now.

Comment (AJTDaedalus) [reply code/app/auth/views.py:107]: I'll take a look

Comment (AJTDaedalus) [reply code/app/auth/views.py:107]: Ok, think I fixed it.  I don't even see a merge so I don't know where that came from at all.  A git pull from master added a lot of stuff I hadn't seen before.

**Comment by LSMachuca** [comment]
It is working now. Both with and without docker

**Pull request merged**